duffle.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "riff",
+    "version": "0.1.0",
+    "schemaVersion": "v1.0.0-WD",
+    "description": "Installs riff",
+    "keywords": [
+        "riff"
+    ],
+    "maintainers": [
+        {
+            "name": "Swapnil Bawaskar",
```

Comment:
the team should maintain the bundle rather than you as an individual. Unless you want to get support questions sent directly to you 😄

Reply:
done!